docs/topics/basic-syntax.md

```diff
@@ -68,6 +68,84 @@ fun main() {
 ```
 {kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="kotlin-basic-syntax-println"}
 
+## Read from the standard input
```

Comment:
@ilya-g had some concerns about placing this paragraph in the Basic syntax.

Reply:
Ok, I'm contacting him.